src/Client.php

```diff
@@ -524,7 +586,7 @@ public function getMouse(): WebDriverMouse
         return new WebDriverMouse($this->webDriver->getMouse(), $this);
     }
 
-    private function createWebDriverByFromLocator(string $locator): WebDriverBy
+    public static function createWebDriverByFromLocator(string $locator): WebDriverBy
```

Comment:
I would move this method to an internal trait or class instead. But marking it `@internal` is maybe enough.